gettsim/transfers/rente.py

```diff
@@ -296,3 +315,62 @@ def ges_rente_regelaltersgrenze(geburtsjahr: int, ges_rente_params: dict) -> flo
         ],
     )
     return out
+
+
+def ges_rente_grenze_altersrente(
+    geburtsjahr: int, geburtsmonat: int, geschlecht: int, ges_rente_params: dict
+) -> float:
+    """Calculates the age, at which a worker is eligible to claim his full pension.
+        Full retirement age (FRA) without deductions. This age is smaller or equal
+        to the regelaltersgrenze (FRA<=NRA) and depends on personal characteristics
+        as gender, insurance duration, health/disability, employment status. Note:
+        This version is just implementing pension  for women. Todo: to be extended
+        for long term insured etc.
+
+    Parameters
+    ----------
+    geburtsjahr
+        See basic input variable :ref:`geburtsjahr <geburtsjahr>`.
+    geburtsmonat
+        See basic input variable :ref:`geburtsmonat <geburtsmonat>`.
+    geschlecht
+        See basic input variable (NEW)
+    ges_rente_params
+        See params documentation :ref:`ges_rente_params <ges_rente_params>`.
+
+    Returns
+    -------
+    """
+    regelrente = piecewise_polynomial(
+        x=geburtsjahr,
+        thresholds=ges_rente_params["regelaltersgrenze"]["thresholds"],
+        rates=ges_rente_params["regelaltersgrenze"]["rates"],
+        intercepts_at_lower_thresholds=ges_rente_params["regelaltersgrenze"][
+            "intercepts_at_lower_thresholds"
+        ],
+    )
+    if geburtsjahr < 1945:
+        x_wom = geburtsjahr + (geburtsmonat - 1) / 12
+    else:
+        x_wom = geburtsjahr
+
+    pension_for_women = piecewise_polynomial(
+        x=x_wom,
+        thresholds=ges_rente_params["altersrente_für_frauen"]["thresholds"],
+        rates=ges_rente_params["altersrente_für_frauen"]["rates"],
+        intercepts_at_lower_thresholds=ges_rente_params["altersrente_für_frauen"][
+            "intercepts_at_lower_thresholds"
+        ],
+    )
+
+    thresholds_m = [regelrente]
```

Comment:
Better to not use the term thresholds as this is already used as an argument for `piecewise_polynomial`.